atd-vzv/src/views/summary/Summary.js

```diff
@@ -30,6 +31,13 @@ const Summary = () => {
   return (
     <Container fluid>
       {/* Create whitespace on sides of view until mobile */}
+      <Row className="px-xs-0 mx-xs-0 px-lg-3 mx-lg-4 mt-4 mb-0">
```

Comment:
Thanks for making this snap out at the same breakpoints as the content below! This would have been easy to overlook! 🕵️

Reply:
👍